src/views/layouts/playerManagement.hbs

```diff
@@ -0,0 +1,191 @@
+            <th style="text-align: center;"><input type="text" id="goalsFilter" onkeyup="filterTable(3)" placeholder="Filter Total Goals"></th>
+            <th style="text-align: center;"></th>
+            <th style="text-align: center;"></th>
+        </tr>
+    </thead>
+        <tbody>
+            {{#each players}}
+                <tr>
+                    <form action="/player/edit/{{_id}}" method="POST" class="edit-form">
+                        <td>
+                            <span class="display-value" style="text-align: left;">{{getTeamName team}}</span>
+                            <select class="form-control edit-input" name="team" required style="display: none;">
+                                {{#each ../teams}}
+                                    <option value="{{_id}}" {{#if (stringeq ../team _id)}}selected{{/if}}>{{name}}</option>
+                                {{/each}}
+                            </select>
+                        </td>
+                        <td style="text-align: left;">
+                            <span class="display-value">{{name}}</span>
+                            <input type="text" class="form-control edit-input" name="name" value="{{name}}" style="display: none;">
+                        </td>
+                        <td style="text-align: center;">
+                            <span class="display-value">{{number}}</span>
+                            <input type="text" class="form-control edit-input" name="number" value="{{number}}" style="display: none;">
+                        </td>
+                        <td style="text-align: center;">
+                            <span class="display-value">{{total_goals}}</span>
+                            <input type="text" class="form-control edit-input" name="total_goals" value="{{total_goals}}" required style="display: none;">
+                        </td>
+                        <td style="text-align: center;">
+                            <span class="display-value">{{rank}}</span>
+                        </td>
+                        <td>
+                            <button type="button" class="btn btn-primary edit-button" onclick="enableEdit(this)">Bearbeiten</button>
+                            <button type="submit" class="btn btn-secondary save-button" style="display: none;">Speichern</button>
+                            <a href="/player/{{_id}}" class="btn btn-info">Tore anzeigen</a>
+                        </td>
+                    </form>
+                </tr>
+            {{/each}}
+        </tbody>
+    </table>
+    <button type="button" class="btn btn-danger mt-3" onclick="resetTotalGoals()">Alle Total Goals zurücksetzen und Tore aus Spieler löschen</button>
+</div>
+
+<script>
+let sortDirections = [true, true, true, true, true]; // Track sort directions for each column
+
+function validateForm() {
```

Comment:
maybe we can put all of these functions into a TableHelper script in a new helper folder making them reusable - just test how much code we can really save i'm not sure if it is worth it

Reply:
i have made a Notion card